plugins/dan.lua

```diff
@@ -0,0 +1,67 @@
+do
+local URL = "http://danbooru.donmai.us"
+local URL_NEW = "/posts.json"
+local URL_POP = "/explore/posts/popular.json"
+
+local scale_day = "?scale=day"
+local scale_week = "?scale=week"
+local scale_month = "?scale=month"
+
+function get_post(url)
+  local b, c, h = http.request(url)
+  local posts = json:decode(b)
+
+  -- bad random - bad magic...
+  math.randomseed( os.time() )
```

Comment:
The `randomseed` is started on https://github.com/yagop/telegram-bot/blob/master/bot/bot.lua#L235
I can merge and remove yours.